tools/license-builder.sh

```diff
@@ -100,4 +100,7 @@ addlicense "brotli" "deps/brotli" "$(cat ${rootdir}/deps/brotli/LICENSE)"
 
 addlicense "HdrHistogram" "deps/histogram" "$(cat ${rootdir}/deps/histogram/LICENSE.txt)"
 
+addlicense "node-heapdump" "src/heap_utils.cc" \
+           "$(curl -sL https://raw.githubusercontent.com/bnoordhuis/node-heapdump/master/LICENSE)"
+
```

Comment:
Can you also re-run the license-builder and include the updated license?

Reply:
I will, yes, once I'm back on an Ubuntu dev machine :-)